### PR TITLE
adding contribfest blog post

### PR DESCRIPTION
**Pull request by codeboten**
Some details still need updating around total contribution numbers.

---

**Preview**: https://deploy-preview-3525--opentelemetry.netlify.app/blog/2023/contribfest-na/

### CODE REVIEW

**Comment by codeboten** [comment]
If folks have better photos of the event, i'm happy to swap these. I recall @jpkrohling & @reese-lee taking some photos at the event

**Review by chalin — COMMENTED** [review]
Nice!

File: content/en/blog/2023/contribfest-2023-na/index.md
Comment (chalin) [inline content/en/blog/2023/contribfest-2023-na/index.md:1]: Drop the stuttered 2023 in the path:

- `content/en/blog/2023/contribfest-2023-na/` --> `content/en/blog/2023/contribfest-na`

Comment (codeboten) [reply content/en/blog/2023/contribfest-2023-na/index.md:1]: Fixed

File: content/en/blog/2023/contribfest-na/index.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+In an effort to bring more contributors to the table, OpenTelemetry maintainers
+hosted the project’s first [Contribfest](https://sched.co/1R2rQ) at KubeCon
+North America 2023. The goal of a Contribfest is to bring people with all levels
+of experience together with community members to work through the steps needed
+to make a contribution to the project. This year,
+[Juraci](https://github.com/jpkrohling/),
+[Anthony](https://github.com/aneurysm9/),
+[Tyler](https://github.com/tylerhelmuth), [Daniel](https://github.com/dyladan),
+and [myself](https://github.com/codeboten) volunteered to host the event and
```
Comment (chalin) [inline content/en/blog/2023/contribfest-na/index.md:28]: 👍🏻 to adding links to GH handles

File: content/en/blog/2023/contribfest-na/index.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+---
+title: OpenTelemetry Contribfest at KubeCon North America 2023
+linkTitle: OpenTelemetry Contribfest at KubeCon North America 2023
```
Comment (chalin) [inline content/en/blog/2023/contribfest-na/index.md:3]:
```suggestion
linkTitle: Contribfest at KubeCon NA
```

**Review by chalin — APPROVED** [review]
LGTM aside from the `linkTitle` change

**Comment by codeboten** [comment]
I found some of the contribfest photos on the CNCF's flickr feed. Updated the images to use those

File: content/en/blog/2023/contribfest-na/index.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
+---
+title: OpenTelemetry Contribfest at KubeCon North America 2023
+linkTitle: Contribfest at KubeCon NA
+date: 2023-11-09
```
Comment (chalin) [inline content/en/blog/2023/contribfest-na/index.md:4]:
```suggestion
date: 2023-11-13
```

**Review by chalin — APPROVED** [review]
Nice photos 👍🏻 
Still looks good to me. I've set the publication date to tomorrow. I'll let @svrnm merge given that he's the blog gate keeper! 😄

**Review by svrnm — APPROVED** [review]
lgtm

**Pull request merged**